pkg/prometheus/config/config.go

```diff
@@ -14,10 +14,15 @@
 
 package config
 
-import "time"
+import (
+	"time"
+
+	"github.com/jaegertracing/jaeger/pkg/config/tlscfg"
+)
 
 // Configuration describes the options to customize the storage behavior.
 type Configuration struct {
-	HostPort       string        `validate:"nonzero" mapstructure:"server"`
-	ConnectTimeout time.Duration `validate:"nonzero" mapstructure:"timeout"`
+	ServerURL      string         `validate:"nonzero" mapstructure:"server"`
```

Comment:
> `validate:"nonzero" mapstructure:"server"`

Where are these being used?  I thought they were introduced when @pavolloffay tried to use OTEL's config mechanism to populate Jaeger configs, but we stopped doing that.

Reply:
If not used, I would remove the annotations

Reply:
The annotations are based on existing example config structs, and I didn't know what they were used for as well tbh; removed.